From 6ad6e71cd11452390453e177afcddf8d3da4963f Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 26 Aug 2020 11:07:54 -0500
Subject: [PATCH 067/120] HSD #18012737990: arm64: dm: fix incorrect watchdog
 node definition

commit ab3c4c1bc6eaddaa7d7723c57e3d44b37712937f from
https://github.com/altera-opensource/linux-socfpga.git

Due to the incorrect clock node definition, watchdog can't initialize
properly during Linux boot process. Therefore the system get reboot in
30 seconds.

Fix the incorrect watchdog node definition.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_dm.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/intel/socfpga_dm.dtsi b/arch/arm64/boot/dts/intel/socfpga_dm.dtsi
index b998a8285ad5..70996ab30cc9 100644
--- a/arch/arm64/boot/dts/intel/socfpga_dm.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_dm.dtsi
@@ -424,7 +424,7 @@
 			reg = <0xffd00200 0x100>;
 			interrupts = <0 117 4>;
 			resets = <&rst WATCHDOG0_RESET>;
-			clock-frequency = <&wdt_clk>;
+			clocks = <&wdt_clk>;
 			status = "disabled";
 		};
 
@@ -433,7 +433,7 @@
 			reg = <0xffd00300 0x100>;
 			interrupts = <0 118 4>;
 			resets = <&rst WATCHDOG1_RESET>;
-			clock-frequency = <&wdt_clk>;
+			clocks = <&wdt_clk>;
 			status = "disabled";
 		};
 
@@ -442,7 +442,7 @@
 			reg = <0xffd00400 0x100>;
 			interrupts = <0 125 4>;
 			resets = <&rst WATCHDOG2_RESET>;
-			clock-frequency = <&wdt_clk>;
+			clocks = <&wdt_clk>;
 			status = "disabled";
 		};
 
@@ -451,7 +451,7 @@
 			reg = <0xffd00500 0x100>;
 			interrupts = <0 126 4>;
 			resets = <&rst WATCHDOG3_RESET>;
-			clock-frequency = <&wdt_clk>;
+			clocks = <&wdt_clk>;
 			status = "disabled";
 		};
 
-- 
2.31.1

